CONTRIBUTING.md

```diff
@@ -0,0 +1,114 @@
+
+## Code of Conduct
+
+The Fluid Community strives to create new community supports, open governance and recognition systems, and
+collaboration techniques that help make our community more open and welcoming. The Gamepad Navigator and everyone
+participating in it is governed by the [Fluid Community's Code of Conduct](https://tinyurl.com/y5q4c6qy). By
+participating, you are expected to uphold this code. Please report unacceptable behavior on the ["#fluid-work" IRC
+channel](https://tinyurl.com/y53ue7d8) or the [fluid-work mailing list](https://tinyurl.com/y4tpf9ty).
+
+## Process/Workflow
+
+The Gamepad Navigator's [source code](src) is hosted on GitHub. All of the code included in a
+[release](https://github.com/fluid-lab/gamepad-navigator/releases) lives in the `master` branch.
+
+The Gamepad Navigator uses a workflow where individual contributors fork the project repository, work in a branch
+created off of `master`, and submit a pull request against the project repository's `master` branch to merge their
+contributions. See the
+[Coding and Commit Standards](https://wiki.fluidproject.org/display/fluid/Coding+and+Commit+Standards) for more clarity
+on the quality and standards of submitted contributions.
+
+### Open an Issue on GitHub
+
+The Gamepad Navigator uses [GitHub issues](https://github.com/fluid-lab/gamepad-navigator/issues) to keep track of the
+open tickets. The branch containing your contribution should be named after the GitHub issue number you're working
+on. For example, branch "GH-28" for issue #28.
+
+GitHub issues should be meaningful and describe the task, bug, or feature in a way that can be understood by the
+community. Opaque or general descriptions should be avoided. If you have a large task that will involve several
+substantial commits, consider breaking it up into subtasks and mention them in the ticket's description. That way, you
+can create multiple pull requests that are easier to review. For example, break issue #28 into two subtasks and make
+a branch for each subtask, such as "GH-28-1" and "GH-28-2".
+
+<!-- TODO: Add an issue and pull request template on GitHub -->
+Go through the following steps to open a new issue/ticket:
+
+1. Search through the existing tickets to see if a similar ticket is already filed. If you find a ticket that seems to
+   match yours, add a comment to the same ticket describing any additional information you have (different context or
+   environment, speculated cause, possible solution, or any other feedback). Otherwise, create a new ticket.
+2. Provide as much relevant detail in the ticket description. If the ticket is a bug report, it should include the
+   following:
+   - Details of the actual behavior.
+   - Instructions for reproducing that bug.
+   - Difference between the actual and expected behavior.
+   - Additional details about the general context.
+3. Besides, more technical information can be added that might help whosoever working on that ticket (see below).
```

Comment:
This needs a little work.  I would use "In addition" instead of "Besides", and avoid "technical" as they may be reporting design issues or mismatches with particular users (such as color or contrast issues).  I'd suggest "In addition, please provide any additional information that might help someone working on the ticket (see below)."